applications/service_brokers/installing-service-catalog.adoc

```diff
@@ -1,19 +1,29 @@
 [id="sb-installing-service-catalog"]
-= Installing the service catalog
+= Installing Service Catalog
 include::modules/common-attributes.adoc[]
 :context: sb-installing-service-catalog
 
 toc::[]
 
 [IMPORTANT]
 ====
-The service catalog is deprecated in {product-title} 4. Equivalent and better
+Service Catalog is deprecated in {product-title} 4. Equivalent and better
 functionality is present in the Operator Framework and Operator Lifecycle
 Manager (OLM).
 ====
 
-// About the service catalog
+[IMPORTANT]
+====
+Starting in {product-title} 4.4, the `service-catalog-controller-manager` and
+`service-catalog-apiserver` cluster Operators are now set to
+`Upgradeable=false`, meaning they will block future cluster upgrades to the next
+minor version, for example 4.5, if they are still installed at that time.
```

Comment:
Suggest breaking this up in 2-3 smaller sentences. For example:

> Starting in {product-title} 4.4, the `service-catalog-controller-manager` and
> `service-catalog-apiserver` cluster Operators are now set to
> `Upgradeable=false`. This means that they will block future cluster upgrades to the next
> minor version, such as 4.5, if they are still installed at that time.

Reply:
Oops meant to do this exact thing! :+1: